### PR TITLE
AT_06.01_007 | Verify of url "People" page

### PR DESCRIPTION
**Pull request by Nzs1**
https://trello.com/c/TO88r7lG/1516-at0601007-verify-of-url-people-page

### CODE REVIEW

**Comment by Nzs1** [comment]
[AT_06.01_007 | Verify of url "People" page](https://trello.com/c/TO88r7lG/1516-at0601007-verify-of-url-people-page)

**Comment by Nzs1** [comment]
[AT_06.01_007 | Verify of url "People" page](https://trello.com/c/TO88r7lG/1516-at0601007-verify-of-url-people-page)

File: cypress/e2e/peoplePage.cy.js [hunk 1]
```diff
@@ -12,4 +13,9 @@ describe("People page", () => {
         cy.get('a[href="/asynchPeople/"]').click();
         cy.get('.jenkins-app-bar__content').should('have.text', pagePeopleText.textPeoplePage);
     })
+
+    it ('AT_06.01_007 | Verify of url "People" page', () => {
+        cy.get('#side-panel').contains(homePage.sidePanelItems[1]).click()
+        cy.url().should('include', '/asynchPeople/')
```
Comment (DariaMas7) [inline cypress/e2e/peoplePage.cy.js:19]: /asynchPeople/ should be in fitures

Comment (Nzs1) [reply cypress/e2e/peoplePage.cy.js]: > /asynchPeople/ should be in fitures

thanks, done

**Review by DariaMas7 — APPROVED** [review]

**Review by byKosta — APPROVED** [review]

**Pull request merged**